src/composables/state-persistor/state-persistor-layer.mapper.ts

```diff
@@ -0,0 +1,44 @@
+import { Layer } from '@/stores/map.store.model'
+import useLayers from '@/composables/layers/layers.composable'
+import useThemes from '@/composables/themes/themes.composable'
+
+const STORAGE_SEPARATOR = '-'
+
+export default function useLayerMapper() {
```

Comment:
Should be a static class, a namespace or a singleton.
We shouldn't use `use` for anything else than `stores` and `composables` IMO